constitution.md

```diff
@@ -0,0 +1,232 @@
+They also resolve disagreements between Reviewers, Developers, Volunteers, and users.
+
+Administrators have all of the rights and responsibilities of Reviewers.
+
+### Reviewers
+Reviewers are selected per project repository, and their primary responsibility is to maintain the health of the code 
+repositories by enforcing a consistent coding style and good software engineering practices. Their second responsibility 
+is to encourage community growth by mentoring enthusiastic contributors.
+
+*Reviewers will be the members who most directly interact with the developer community, so it is of primary importance that 
+they encourage a welcoming atmosphere.*
+
+Reviewers are the only members who have write-privileges to their repository, so they must be active regularly such that 
+development will not stall on their reviews. The procedure for becoming a Reviewer is outlined in the following Developers 
+section.
+
+Reviewers strictly subsume Developers (e.g. they also have all the rights and responsibilities of Developers).
+
+### Developers
+Developers are the most common membership type, and is an entry point for enthusiastic community members. They have voting
+rights for high-level directives as set by the administrators of a given project.
+
+Anyone can become a developer by signing up and *signing the CLA*. In exchange for voting rights, a developer must commit to 
+5 hours per month of development work on any Chisel project. This work can include:
+* creating pull requests
+* reviewing pull requests
+* submitting issues
+* discussing issues
+* answering Stack Overflow questions
+* attending meetings
+* external contributions (registered on website via submissions)
+  * writing blogs
+  * additional documentation
+* red lighting/green lighting releases
+
+Developers who have been inactive for six months or more will be removed from the group.
+
+A developer interested in becoming a reviewer should have a wide body of commits and begin reviewing submitted pull 
+requests to practice and build up a body of reviews, both to demonstrate their expertise but also to increase their 
+familiarity with the repository. These pull-requests will also require reviews from a Reviewer. Once a developer has 
+had over 10 reviews on a project, they can submit a request to the administrators to become a Reviewer for that project.
+
+Developers can contribute feedback by testing scheduled releases and fixes, and these members are critical to ensuring 
+Chisel's backwards compatibility, as well as discovering bugs before a release. 
+
+### Volunteers
+Volunteers are people who want to contribute to a Chisel project, but do not commit to a regular contribution. They do 
+not have voting rights, and no registration is required. Volunteers may attend development meetings. Volunteers may 
+contribute through PR’s to Chisel repositories, and are encouraged to review and comment but they cannot accept PR requests. It is hoped that volunteers will become developers.
```

Comment:
Very minor nit, but I think it is incorrect to use an apostrophe with an initialism. `s/PR's/PRs/`. Other examples below.